### ISSUE TITLE
Authorisation

### ISSUE THREAD

**Issue by jackherizsmith** [fac19/Bechdel-Beyond-backend#59]
Was surprised to find authorisation inside the one express route that needs it - nice. This might need to be abstracted back to middleware in the case of future authorised-only routes (e.g. user settings?)

**Comment by akomiqaia** [comment]
You are absolutely right and we intend to when we have reached a stage where users can edit their setting :laughing:
